model/src/main/kotlin/config/LicenseFilePatterns.kt

```diff
@@ -40,14 +42,15 @@ data class LicenseFilePatterns(
      * A set of globs that match files that often contain the license of a project, but that are no license files and
      * are therefore not contained in [licenseFilenames].
      */
-    val rootLicenseFilenames: Set<String>
+    @ConfigAlias("rootLicenseFilenames")
+    val otherLicenseFilenames: Set<String>
```

Comment:
Does the de-serialization of the config fail in case the alias is omit?
If yes (it does not go unnoticed), shall we just make this a breaking change to avoid having this historical thing in the code?

Reply:
> Does the de-serialization of the config fail in case the alias is omit?

Yes.

> If yes (it does not go unnoticed), shall we just make this a breaking change to avoid having this historical thing in the code?

I'd be fine with that, as probably not many users have customized this anyway. Doing so (i.e. not using `@ConfigAlias`) would also work-around https://github.com/sksamuel/hoplite/issues/455.